src/runtime_src/driver/xclng/drm/xocl/mgmtpf/mgmt-core.c

```diff
@@ -483,9 +487,15 @@ static int xclmgmt_setup_msix(struct xclmgmt_dev *lro)
 	 * cover its own usr vectors. So, only the last chunk of the table will
 	 * ever be used for mgmt pf.
 	 */
-	lro->msix_user_start_vector = XOCL_READ_REG32(lro->core.intr_bar_addr +
-		XCLMGMT_INTR_USER_VECTOR) & 0x0f;
-	total = lro->msix_user_start_vector + XCLMGMT_MAX_USER_INTR;
+	if (dev_info->flags & XOCL_DSAFLAG_FIXED_INTR) {
```

Comment:
The comment above does not apply to fixed intr case. please move it into the else clause.

Reply:
done